### DISCUSSION TITLE
Provided device doesn't support required NVENC features

### DISCUSSION THREAD

**Discussion by UrsusLvovich** [LizardByte/community#341] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Install Sunshine v0.21.0
Open the UI, get this error
![image](https://github.com/LizardByte/Sunshine/assets/38516315/81cb1ddc-c00a-45f3-a423-6ffe4f5476c8)

### Expected Behavior

Be able to install Sunshine and run no  problems on modern hardware.

### Additional Context

Host OS: Pop OS 22.04LTS
CPU: AMD Ryzen 7 3800X
GPU: Nvidia RTX 3080
Sunshine Version v0.21.0

I did a little digging and it seems that the RTX 3080 doesn't support av1_nvenc encoding. Does this mean I can't use Sunshine with my 3080?
https://www.reddit.com/r/AV1/comments/160tsq1/av1_software_encoding_on_rtx_3080_efficiency_and/

### Host Operating System

Linux

### Operating System Version

Pop OS 22,04 LTS

### Architecture

64 bit

### Sunshine commit or version

0.21.0

### Package

Linux - flatpak

### GPU Type

Nvidia

### GPU Model

RTC 3080

### GPU Driver/Mesa Version

545.29.02

### Capture Method (Linux Only)

PipeWire

### Config

```Shell
my ~/.config/sunshine/sunshine.conf file is empty
```

### Relevant log output

```shell
[2023:11:16:17:33:46]: Warning: [av1_nvenc @ 0x5557c26f0b40] Codec not supported
[2023:11:16:17:33:46]: Fatal: [av1_nvenc @ 0x5557c26f0b40] Provided device doesn't support required NVENC features
[2023:11:16:17:33:46]: Error: Could not open codec [av1_nvenc]: Function not implemented
```

**Comment by ReenigneArcher** [comment]
We use GitHub issues specifically for "actionable bug reports"... your card doesn't support AV1 encoding.

Sunshine will still work though.

**Reply by UrsusLvovich** [reply]
how would I get sunshine to work? make my cpu do the av1 encoding?

**Reply by ReenigneArcher** [reply]
It will use a different encoder... did you actually try streaming?

**Reply by UrsusLvovich** [reply]
I downloaded moonlight on my Steam deck but didn't see my host PC available

**Comment by UrsusLvovich** [comment, marked as answer]
So I guess discovery didn't work, but I got a tip to input my PC public IP in manually in Moonlight/Sunshine. That allowed me to view my desktop on my steamdeck. Thanks for the help
